hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenAirdropHandler.java

```diff
@@ -306,7 +306,7 @@ final var record = createPendingAirdropRecord(pendingId, null);
      * the credits for the receivers.
      * @param fungibleAmounts the fungible airdrop amounts
      * @param sender the sender account id
-     * @param isApproval if the airdrop transfer is sent with an approval
+     * @param isApproval should be false for airdrop
```

Comment:
probably just add enum `AllowanceStrategy` 
 with `ALLOWANCES_ALLOWED` and `ALLOWANCES_REJECTED` for it to be readable ?

Reply:
Will do in different PR.